packages/tenant-process/src/services/tenantService.ts

```diff
@@ -175,22 +171,6 @@ async function retrieveAgreement(
   return agreement;
 }
 
-async function retrieveActiveDelegation(
```

Comment:
Removed in order to not cause confusion with other retrieves; we usually define a retrieve when we assert that the object exists otherwise we throw, this one instead just calls the method in the readModel